python_modules/dagster/dagster/core/execution/plan/plan.py

```diff
@@ -907,8 +906,12 @@ def build_memoized_plan(
                 if not io_manager.has_output(context):
                     unmemoized_step_keys.add(step_output_handle.step_key)
 
+        if selected_step_keys is not None:
+            step_keys_to_execute = list(unmemoized_step_keys & set(selected_step_keys))
```

Comment:
nit: maybe a comment here just for extra clarification for dumb dumbs like me

Reply:
👍